gradle/fips.gradle

```diff
@@ -36,8 +36,8 @@ if (BuildParams.inFipsJvm) {
       fipsPolicy = new File(fipsResourcesDir, "fips_java_bcjsse_11.policy")
     }
     File fipsTrustStore = new File(fipsResourcesDir, 'cacerts.bcfks')
-    def bcFips = dependencies.create('org.bouncycastle:bc-fips:1.0.2.1')
-    def bcTlsFips = dependencies.create('org.bouncycastle:bctls-fips:1.0.12.2')
+    def bcFips = dependencies.create('org.bouncycastle:bc-fips:2.0.0')
```

Comment:
Do you know if dependabot is monitoring this file? I wasn't aware that this file also had dependencies declared here.